### PR TITLE
Add a few references in the further resources page

### PR DESCRIPTION
**Pull request by zliu41**

### CODE REVIEW

File: doc/docusaurus/docs/delve-deeper/further-resources.md [hunk 1]
```diff
@@ -4,18 +4,28 @@ sidebar_position: 40
...
+
+9. Polina Vinogradova, Orestis Melkonian, Philip Wadler, Manuel Chakravarty, Jacco Krijnen Michael Peyton Jones, James Chapman, and Tudor Ferariu. [Structured Contracts in the EUTxO Ledger Model](https://iohk.io/en/research/library/papers/structured-contracts-in-the-eutxo-ledger-model/). In *5th International Workshop on Formal Methods for Blockchains (FMBC)*, 2024.
+
+10. Manuel MT Chakravarty, Simon Thompson, and Philip Wadler. [Functional smart contracts on cardano](https://www.youtube.com/watch?v=MpWeg6Fg0t8).
+
+11. IOHK. Formal specification of the Plutus Core language. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
+
+12. IOHK. Plutus platform technical report. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
+
+13. Michael Peyton Jones and Jann Müller. [The plutus platform](https://www.youtube.com/watch?v=usMPt8KpBeI).
```
Comment (effectfully) [inline doc/docusaurus/docs/delve-deeper/further-resources.md:31]: This is probably horribly outdated?

File: doc/docusaurus/docs/delve-deeper/further-resources.md [hunk 2]
```diff
@@ -4,18 +4,28 @@ sidebar_position: 40
...
+8. Polina Vinogradova and Orestis Melkonian. [Message-passing in the Extended UTxO Ledger Model](https://iohk.io/en/research/library/papers/message-passing-in-the-extended-utxo-ledger-model/). In *8th Workshop on Trusted Smart Contracts (WTSC)*, 2024.
+
+9. Polina Vinogradova, Orestis Melkonian, Philip Wadler, Manuel Chakravarty, Jacco Krijnen Michael Peyton Jones, James Chapman, and Tudor Ferariu. [Structured Contracts in the EUTxO Ledger Model](https://iohk.io/en/research/library/papers/structured-contracts-in-the-eutxo-ledger-model/). In *5th International Workshop on Formal Methods for Blockchains (FMBC)*, 2024.
+
+10. Manuel MT Chakravarty, Simon Thompson, and Philip Wadler. [Functional smart contracts on cardano](https://www.youtube.com/watch?v=MpWeg6Fg0t8).
+
+11. IOHK. Formal specification of the Plutus Core language. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
+
+12. IOHK. Plutus platform technical report. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
```
Comment (effectfully) [inline doc/docusaurus/docs/delve-deeper/further-resources.md:29]: This is definitely outdated ([here](https://input-output-rnd.slack.com/archives/G01EE4NQ9U0/p1720083189564909?thread_ts=1719917481.029889&cid=G01EE4NQ9U0)'s a Slack comment by Michael).

File: doc/docusaurus/docs/delve-deeper/further-resources.md [hunk 3]
```diff
@@ -4,18 +4,28 @@ sidebar_position: 40
...
 
-8. Michael Peyton Jones and Jann Müller. [The plutus platform](https://www.youtube.com/watch?v=usMPt8KpBeI). 
+8. Polina Vinogradova and Orestis Melkonian. [Message-passing in the Extended UTxO Ledger Model](https://iohk.io/en/research/library/papers/message-passing-in-the-extended-utxo-ledger-model/). In *8th Workshop on Trusted Smart Contracts (WTSC)*, 2024.
+
+9. Polina Vinogradova, Orestis Melkonian, Philip Wadler, Manuel Chakravarty, Jacco Krijnen Michael Peyton Jones, James Chapman, and Tudor Ferariu. [Structured Contracts in the EUTxO Ledger Model](https://iohk.io/en/research/library/papers/structured-contracts-in-the-eutxo-ledger-model/). In *5th International Workshop on Formal Methods for Blockchains (FMBC)*, 2024.
+
+10. Manuel MT Chakravarty, Simon Thompson, and Philip Wadler. [Functional smart contracts on cardano](https://www.youtube.com/watch?v=MpWeg6Fg0t8).
+
+11. IOHK. Formal specification of the Plutus Core language. Technical Report, IOHK, 2019. Available at [https://github.com/IntersectMBO/plutus](https://github.com/IntersectMBO/plutus).
```
Comment (effectfully) [inline doc/docusaurus/docs/delve-deeper/further-resources.md:27]: 2019? Insert a link pointing to the actual spec?

**Review by effectfully — APPROVED** [review]
:+1:

**Review by Unisay — APPROVED** [review]

**Pull request merged**